modules/transport/http/src/org/apache/axis2/transport/http/util/HTTPProxyConfigurationUtil.java

```diff
@@ -154,6 +156,20 @@ public static void configure(MessageContext messageContext,
             proxyPort = Integer.parseInt(port);
         }
 
+        String userName = System.getProperty(HTTP_PROXY_USER);
+        if (userName != null) {
+            proxyUser = userName;
+        }
+
+        String password = System.getProperty(HTTP_PROXY_PASSWORD);
+        if (password != null) {
+            proxyPassword = password;
+        }
+
+        if (proxyUser != null && proxyPassword != null) {
```

Comment:
this makes proxyCredentials assignment twice even when system properties are not defined. This will break NTCredentials case see L141

Reply:
Done